src/sentry/interfaces/stacktrace.py

```diff
@@ -411,8 +411,8 @@ def get_hash(self):
                 output.append('<function>')
             else:
                 output.append(remove_function_outliers(self.function))
-        elif self.lineno is not None:
-            output.append(self.lineno)
+            if self.lineno is not None:
```

Comment:
Thank you for comment.

I think it won't break the case both a context line and lineno are present. But breaks the case both function and lineno are present.

I organized my proposal as table below. Is this kind of change acceptable?
If not I will close this PR and seek other solution.

Before change

context_line present? |  function present? | lineno present? | output |
--- | --- | --- | ---
y | y | y | context_line |
y | n | y | lineno |
y | y | n | context_line |
n | y | y | function |
n | n | y | lineno |
n | y | n | function |

After change

context_line present? |  function present? | lineno present? | output |
--- | --- | --- | ---
y | y | y | context_line |
y | n | y | lineno |
y | y | n | context_line |
n | y | y | function and lineno |
n | n | y | lineno *(This PR code do not support this case...)* |
n | y | n | function |

※ Assume symbol is not present in all case. If symbol is present it will be in output instead of function or lineno.